src/views/Profile/AuthorsList/AuthorsDeleteBtn.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { Tooltip } from '@material-ui/core';
+import i18n, { langTokens } from '../../../locales/localizationInit';
+import AuthorsDeletingDialog from './AuthorsDeletingDialog';
+
+interface IAuthorsDeleteBtnProps {
+  id: number;
+  fullName: string;
+  isAllowedToDelete?: boolean;
+}
+
+const AuthorsDeleteBtn: React.FC<IAuthorsDeleteBtnProps> = (props) => {
+  const tooltipTitle = i18n.t(langTokens.admin.removeAuthorWarning);
+  const { id, fullName, isAllowedToDelete } = props;
+
+  const [openTooltip, setOpenTooltip] = useState(false);
+
+  const handleOpenTooltip = () => {
+    setOpenTooltip(true);
+  };
+  const handleCloseTooltip = () => {
+    setOpenTooltip(false);
+  };
+
+  return (
+    <>
+      <Tooltip
```

Comment:
I think we don't need separate state here. it looks like there is no _single_ prop to disable tooltip from appearing, but we can do it like:

```
<Tooltip
  disableFocusListener={isAllowedToDelete}
  disableHoverListener={isAllowedToDelete}
  disableTouchListener={isAllowedToDelete}
```

Reply:
or even with passing empty `title`, api docs say that in this case tooltip will not be shown:

```
<Tooltip
  title={isAllowedToDelete ? null : tooltipTitle}
```

Reply:
changed to the first version described in comment